lib/builder-manager/src/utils/template.ts

```diff
@@ -72,6 +73,12 @@ export const renderHTML = async (
       FEATURES: JSON.stringify(await features, null, 2),
```

Comment:
We seem to `apply('features')` and await it several times unnecessarily throughout the code base, as we already do this:

https://github.com/storybookjs/storybook/blob/c57358f212f0d5fcb5c5c58223f75204385af56a/lib/core-server/src/build-dev.ts#L95-L103

The same may be true for `logLevel` and `refs`, I'm not sure. I don't pretend to understand this code well enough to say.

Reply:
Evaluating the preset value once, and passing down the value might increase perf by a tiny bit, but at the cost of complexity.

Apply-ing a preset multiple times, is the intended behavior. But what we should do, at some point, is to enable memoization on it, so calling it multiple times with the same options, gets cached.